qa/L0_perf_analyzer_capi/test.sh

```diff
@@ -0,0 +1,396 @@
+fi
+if [ $(cat $CLIENT_LOG | grep "${ERROR_STRING}" | wc -l) -ne 0 ]; then
+    cat $CLIENT_LOG
+    echo -e "\n***\n*** Test Failed\n***"
+    RET=1
+fi
+
+
+$PERF_ANALYZER -v -m graphdef_int32_int32_int32 -t 1 -p2000 -b 1 \
+--service-kind=triton_local --model-repo=$DATADIR --library-name=$SERVER_LIBRARY_PATH >$CLIENT_LOG 2>&1
+if [ $? -ne 0 ]; then
+    cat $CLIENT_LOG
+    echo -e "\n***\n*** Test Failed\n***"
+    RET=1
+fi
+if [ $(cat $CLIENT_LOG | grep "${ERROR_STRING}" | wc -l) -ne 0 ]; then
+    cat $CLIENT_LOG
+    echo -e "\n***\n*** Test Failed\n***"
+    RET=1
+fi
+
+
+#Testing that async does NOT work
+
+$PERF_ANALYZER -v -m graphdef_int32_int32_int32 -t 1 -p2000 -b 1 -a \
+--service-kind=triton_local --model-repo=$DATADIR --library-name=$SERVER_LIBRARY_PATH >$CLIENT_LOG 2>&1
+if [ $? -eq 0 ]; then
+    cat $CLIENT_LOG
+    echo -e "\n***\n*** Test Failed\n***"
+    RET=1
+fi
+
+
+#Testing that shared memory does not work
+for SHARED_MEMORY_TYPE in system cuda; do
+    set +e
+    $PERF_ANALYZER -v -m graphdef_int32_int32_int32 -t 1 -p2000 -b 1 --shared-memory=$SHARED_MEMORY_TYPE \
+    --service-kind=triton_local --model-repo=$DATADIR --library-name=$SERVER_LIBRARY_PATH >$CLIENT_LOG 2>&1
+    if [ $? -ne 1 ]; then
+        cat $CLIENT_LOG
+        echo -e "\n***\n*** Test Failed\n***"
+        RET=1
+    fi
+    
+done
+
+# Testing --request-rate-range does NOT work
+
+$PERF_ANALYZER -v -m graphdef_int32_int32_int32 --request-rate-range 1000:2000:500 -p1000 -b 1 \
+--service-kind=triton_local --model-repo=$DATADIR --library-name=$SERVER_LIBRARY_PATH >$CLIENT_LOG 2>&1
+if [ $? -eq 0 ]; then
+    cat $CLIENT_LOG
+    echo -e "\n***\n*** Test Failed\n***"
+    RET=1
+fi
+
+
+# Testing with inception model
+
+$PERF_ANALYZER -v -m inception_v1_graphdef -t 1 -p2000 -b 1 \
+--service-kind=triton_local --model-repo=$DATADIR --library-name=$SERVER_LIBRARY_PATH >$CLIENT_LOG 2>&1
+if [ $? -ne 0 ]; then
+    cat $CLIENT_LOG
+    echo -e "\n***\n*** Test Failed\n***"
+    RET=1
+fi
+if [ $(cat $CLIENT_LOG | grep "${ERROR_STRING}" | wc -l) -ne 0 ]; then
+    cat $CLIENT_LOG
+    echo -e "\n***\n*** Test Failed\n***"
+    RET=1
+fi
+
+# Testing with resnet50 models with large batch sizes
+
+$PERF_ANALYZER -v -m inception_v1_graphdef -t 2 -p2000 -b 64 \
+--service-kind=triton_local --model-repo=$DATADIR --library-name=$SERVER_LIBRARY_PATH >$CLIENT_LOG 2>&1
+if [ $? -ne 0 ]; then
+    cat $CLIENT_LOG
+    echo -e "\n***\n*** Test Failed\n***"
+    RET=1
+fi
+if [ $(cat $CLIENT_LOG | grep "${ERROR_STRING}" | wc -l) -ne 0 ]; then
+    cat $CLIENT_LOG
+    echo -e "\n***\n*** Test Failed\n***"
+    RET=1
+fi
+
+# Test perf client behavior on different model with different batch size
+for MODEL in graphdef_nobatch_int32_int32_int32 graphdef_int32_int32_int32; do
+    # Valid batch size
+    $PERF_ANALYZER -v  -m $MODEL -t 1 -p2000 -b 1 \
+    --service-kind=triton_local --model-repo=$DATADIR --library-name=$SERVER_LIBRARY_PATH >$CLIENT_LOG 2>&1
```

Comment:
Reminder to modify triton_local to triton_c_api as based on suggestions on client side PR

Reply:
Still needs this to be fixed..